bundles/org.openhab.binding.lgtvserial/src/main/java/org/openhab/binding/lgtvserial/internal/handler/LgTvSerialHandler.java

```diff
@@ -105,6 +105,7 @@ public LgTvSerialHandler(Thing thing, SerialCommunicatorFactory factory, SerialP
     @Override
     public synchronized void initialize() {
         String portName = (String) getThing().getConfiguration().get("port");
+        int refreshInterval = ((BigDecimal) getThing().getConfiguration().get("refreshInterval")).intValue();
```

Comment:
Now that `getThing().getConfiguration()` is being called three times after each other, you could consider caching it in a local variable.